src/color-inline/README.md

```diff
@@ -28,4 +34,38 @@ Invalid color:
 
 ```html
 <color-inline>foobar</color-inline>
-```
\ No newline at end of file
+```
+
+## Reference
+
+### Slots
+
+| Name | Description |
+|------|-------------|
+| (default) | The element's main content—the color to be shown. Placed next to the color swatch. |
+
+### Attributes & Properties
+
+| Attribute | Property | Property type | Default value | Description |
+|-----------|----------|---------------|---------------|-------------|
+| `color` | `color` | `Color` &#124; `null` | - | The current color value. `null` for invalid colors. |
+| `value` | `value` | `string` &#124; `undefined` | - | The current value. `undefined` if the `value` attribute is not set. |
```

Comment:
It’s not helpful to document `value` as "the current value", that means nothing. It's as informative as the old `i++; // increment i` joke. 

Explain that it’s the textual form of the color, and will have a value even if the color is invalid.